### DISCUSSION TITLE
Facing problem in lesson 9 practice set

### DISCUSSION THREAD

**Discussion by Gloryjaw** [Cyfrin/foundry-full-course-cu#1558] [Q&A] [answered]
As you probably know that this practice set will require you to guess the right number. I am doing it through a contract. First I create a number which will be the right guess in a contract then I call solve challenge with that number as an argument from that contract. I have also created interface which can receive erc721 token and then transfer it to my account ( I don't know about tokens and all, I asked gpt because it will be taught in lesson 10 and I just completed lesson 9). Everything is working fine, the nft gets minted and transferred to my account, however, it doesn't show in etherscan as a transaction like it did for other nfts. I don't know why that is happening.

**Comment by EngrPips** [comment, marked as answer]
Hello @Gloryjaw , If you minted on `sepolia testnet` and you have the `Nft` already in your wallet, i would say you shouldn't worry too much if the transaction didn't index on etherscan as soon as you expect. as test net can be slow sometimes as they run at the goodness of people heart.

**Reply by Gloryjaw** [reply]
Hi, I was reading the book called mastering ethereum and it talks about internal transactions. They are the transactions which are between one contract to other contract and they are never serialised, I think that's probably the reason why my transaction is not showing since I minted nfts with the help of a contract and not with eoa. I hope I am not wrong 😅

**Reply by EngrPips** [reply]
Yeah this makes sense.
